packages/js/package.json

```diff
@@ -31,7 +31,8 @@
   "private": true,
   "scripts": {
     "start": "pnpm run build -- --watch --sourcemap",
-    "build": "tsup && pnpm run post:build",
+    "build": "tsup && pnpm run build:umd && pnpm run post:build",
+    "build:umd": "webpack --config webpack.config.cjs",
```

Comment:
build UMD with Webpack

Reply:
https://linear.app/novu/issue/COM-58/give-a-second-chance-to-tsup-to-build-the-umd-files